Lib/unittest/test/test_case.py

```diff
@@ -596,6 +596,48 @@ def testShortDescriptionWithMultiLineDocstring(self):
                  'Tests shortDescription() for a method with a longer '
                  'docstring.')
 
+    @unittest.skipIf(
+        sys.flags.optimize >= 2,
+        "Docstrings are omitted with -O2 and above")
+    def testShortDescriptionWithSurroundingSpaceOneLineDocstring(self):
```

Comment:
Sorry, not sure I understand. You appear to be saying that we should use the same docstring as for the `testShortDescriptionWithOneLineDocstring` test but with extra white space. However it is a different test - doesn't that mean it needs a different doc string to describe how it is different from the previous docstring?